core/src/main/java/com/redhat/lightblue/client/request/AbstractLightblueDataExecutionRequest.java

```diff
@@ -0,0 +1,28 @@
+package com.redhat.lightblue.client.request;
+
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.redhat.lightblue.client.Execution;
+
+public abstract class AbstractLightblueDataExecutionRequest extends AbstractLightblueDataRequest {
```

Comment:
Made appendExecutionToJson private. Execution bits are added in AbstractLightblueDataExecutionRequest.getBodyJson() now.